app/src/main/java/sample/okta/android/dashboard/DashboardViewModel.kt

```diff
@@ -119,7 +119,7 @@ internal class DashboardViewModel(private val credentialTagNameValue: String?) :
         viewModelScope.launch {
             val idToken = credential.token?.idToken ?: return@launch
             when (
-                val result = CredentialBootstrap.oidcClient.createWebAuthenticationClient().logoutOfBrowser(
+                val result = WebAuthenticationClient().logoutOfBrowser(
```

Comment:
Is the recommended pattern to create a client instance every time? Or will there be a shared singleton instance of the default WebAuthentication class like Swift's [WebAuthentication.shared](https://okta.github.io/okta-mobile-swift/development/webauthenticationui/documentation/webauthenticationui/webauthentication/shared) static property. 

The Swift SDK did it this way so listeners/delegates can be added to the shared instance, and would persist across interactions with it.

Reply:
There is one (internally set) listener that the WebAuthentication class references, but that itself is Singleton. Users get the result of WebAuthentication, while we handle the listeners internally. Since listeners can't be set by the users for WebAuthentication, I don't see much value in adding a shared static property here.